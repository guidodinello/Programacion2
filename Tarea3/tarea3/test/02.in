# Prueba de esVacioBinario, insertarEnBinario, raiz, izquierdo, derecho, buscarSubarbol
buscarSubarbol 1
esVacioBinario
insertarEnBinario (1,1.1)
esVacioBinario
raiz
buscarSubarbol 2
buscarSubarbol 1
insertarEnBinario (2,2.2)
insertarEnBinario (3,3.3)
izquierdo
derecho
buscarSubarbol 4
buscarSubarbol 1
buscarSubarbol 2
buscarSubarbol 3
Fin
