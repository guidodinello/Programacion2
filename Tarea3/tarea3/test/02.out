1>#  Prueba de esVacioBinario, insertarEnBinario, raiz, izquierdo, derecho, buscarSubarbol.
2>sub es vacío.
3>Vacio.
4>Insertado.
5>No vacío.
6>(1,1.10)
7>sub es vacío.
8>(1,1.10)
9>Insertado.
10>Insertado.
11>izquierdo de b es vacío.
12>(2,2.20)
13>sub es vacío.
14>(1,1.10)
15>(2,2.20)
16>(3,3.30)
17>Fin.
